modules/counter/counter.block.ini

```diff
@@ -14,6 +14,12 @@ description: Rising edge ticks the counter up/down by STEP
 type: bit_mux
 description: Up/Down direction (0 = Up, 1 = Down)
 
+[OUT_MODE]
+type: param enum
+description: Counter OUT update mode (on internal counter value change or on ENABLE falling edge)
+0: On_change
+1: On_disable
```

Comment:
Think I'd probably prefer enum strings with spaces to be more consistent with other enumeration options: `On Change` and `On Disable`.

Reply:
good with On-Change + On-Disable like LUT

Reply:
That sounds fine to me, we have a mix of standards anyway